include/ignition/common/Time.hh

```diff
@@ -52,24 +52,32 @@ namespace ignition
       };
 
       /// \brief Constructors
-      public: Time();
+      public: [[deprecated("Do not use this class, use std::chrono")]] Time();
```

Comment:
We shouldn't introduce deprecations in a released major. We can add the new functions here, but let's add the deprecations to `master`.

Also, please use `IGN_DEPRECATED(3)` instead of `[[deprecated]]`.

Reply:
`master` is version 4, so `IGN_DEPRECATED(4)`?